### DISCUSSION TITLE
Redid the config and now have a rouge stack

### DISCUSSION THREAD

**Discussion by RiffyDivine** [louislam/dockge#149] [Ask for Help]
So I made a mistake in my config file and didn't notice. I brought it down and fixed it before bring it back up. Now I have a stack that is listed as not managed by Dockge. I am unable to find it or delete it. Any idea where it ended up or should I just nuke and start over? My dockge stack folder was pointed to the wrong folder but when I went into it thinking it just made the folder in the /home/ it's not in there.

**Comment by louislam** [comment]
`docker compose ls`

You can find where are they.

**Comment by jtodaro** [comment]
> So I made a mistake in my config file and didn't notice. I brought it down and fixed it before bring it back up. Now I have a stack that is listed as not managed by Dockge. I am unable to find it or delete it. Any idea where it ended up or should I just nuke and start over? My dockge stack folder was pointed to the wrong folder but when I went into it thinking it just made the folder in the /home/ it's not in there.

I actually did the same thing.

> `docker compose ls`
> 
> You can find where are they.

Unfortunately it doesn't look like that works, as the "rogue" stack can't be started/stopped (shows exited and "This stack is not managed by Dockge.").

It almost acts like the "rogue" stack is stored in a db list somewhere.

**Reply by jtodaro** [reply]
Actually figured it out. Had to recreate original named stack dir, then create a mock compose.yaml. I was then able to delete the "rogue" stack.

**Reply by RiffyDivine** [reply]
I just used docker prune when I did it. Sorry I should have come back and posted this awhile ago. https://docs.docker.com/engine/reference/commandline/system_prune/
